### PR TITLE
Add Access log fields for Proxy Protocol context

### PR DESCRIPTION
**Pull request by sudheerv**

### CODE REVIEW

File: proxy/logging/LogAccess.cc [hunk 1]
```diff
@@ -1315,6 +1315,61 @@ LogAccess::marshal_version_build_number(char *buf)
...
+      version_str = "V1";
+      break;
+    case NetVConnection::ProxyProtocolVersion::V2:
+      version_str = "V2";
+      break;
+    case NetVConnection::ProxyProtocolVersion::UNDEFINED:
+    default:
+      version_str = "UNK";
+      break;
+    }
```
Comment (randall) [inline proxy/logging/LogAccess.cc:1336]: This this be '-'? That would seem to follow the if something is not set, log `-`. Or is UNDEFINED different than not set?

Comment (sudheerv) [reply proxy/logging/LogAccess.cc]: > This this be '-'? That would seem to follow the if something is not set, log `-`. Or is UNDEFINED different than not set?

Ah agree. `-` is indeed better.

**Comment by randall** [comment]
Resubmitted as #6714

**Comment by zwoop** [comment]
Please remember to remove Milestone and Projects from PRs that are closed without merging.

**Pull request closed without merging**